c10/core/TensorImpl.h

```diff
@@ -2208,11 +2164,25 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
     // Customizable strides behavior, e.g., sparse tensor,
     // mkldnn tensor.
     //
-    // Can override: strides(), is_contiguous()
+    // Can override:
+    //   - strides()
+    //   - is_contiguous()
+    //   - empty_tensor_restride()
+    //   - refresh_contiguous()
+    //   - set_sizes_and_strides()
```

Comment:
This one seems a bit weird, if you have customizable strides behavior but not sizes behavior, why let users customize BOTH sizes and strides?